espnet2/asr/encoder/conformer_encoder.py

```diff
@@ -123,10 +126,10 @@ def __init__(
         elif pos_enc_layer_type == "scaled_abs_pos":
             pos_enc_class = ScaledPositionalEncoding
         elif pos_enc_layer_type == "rel_pos":
-            assert selfattention_layer_type == "rel_selfattn"
+            assert selfattention_layer_type in ["rel_selfattn", "lf_selfattn"]
             pos_enc_class = RelPositionalEncoding
         elif pos_enc_layer_type == "legacy_rel_pos":
-            assert selfattention_layer_type == "legacy_rel_selfattn"
+            assert selfattention_layer_type in ["rel_selfattn", "lf_selfattn"]
```

Comment:
The original code is `legacy_rel_selfattn`, but here it is changed to `rel_selfattn`.
If the positional encoding layer is `legacy`, I think the self-attention layer should also be `legacy`?

Reply:
Will fix this